### PR TITLE
Feature/power up update

### PR DESCRIPTION
**Pull request by VonIobro**
Lots of changes here:

- [x] use ExchangeDialog instead of TransferDialog  to allow for calc estimation
- [x] add min amounts to ensure transaction will be successful
- [x] show estimations for powerup and powerdown
- [x] ensure that player cannot not request more powerup than possible
- [x] test for `minimumPowerUpSizeBabz()` from https://github.com/acebusters/economy/pull/47
- [x] gracefully switch between default of 10,000 babz min to value from minimumPowerUpSizeBabz(), if available

### CODE REVIEW

**Comment by sunify** [comment]
Everything works fine for me :-)

**Comment by sunify** [comment]
Resolves #676

**Comment by VonIobro** [comment]
@sunify This should be ready to merge after shippable tests^^

File: app/containers/Dashboard/index.js [hunk 1]
```diff
@@ -218,6 +218,9 @@ class DashboardRoot extends React.Component {
     this.power.downtime.call();
     this.power.totalSupply.call();
     this.power.activeSupply.call();
+    if (typeof this.power.minimumPowerUpSizeBabz === 'function') {
```
Comment (VonIobro) [inline app/containers/Dashboard/index.js]: After this is merged, I will create an issue to remind us to remove this check after the new ABI contracts are deployed successfully. (Lines 221 and 397)

**Pull request merged**